src/app/service/event.service.ts

```diff
@@ -0,0 +1,25 @@
+
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { EventRequest } from '../entity/EventRequest';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class EventService {
+  private addUrl = 'http://localhost:8080/api/v1/events/add';
```

Comment:
It's a good idea to move the base URL (in this case `http://localhost:8080`) to the `environment.ts` file, so it can be easily changed